lib/fs.js

```diff
@@ -221,19 +191,17 @@ fs.Stats.prototype.isSocket = function() {
 });
 
 fs.access = function(path, mode, callback) {
+  callback = makeCallback(arguments[arguments.length - 1]);
```

Comment:
@jasnell Perhaps I can traverse the `arguments` backwards and see if there is atleast one function object passed?